public/index.html

```diff
@@ -1,6 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-148748595-1"></script>
+    <script>
+      if ('%NODE_ENV%' === 'production') {
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-148748595-1');
```

Comment:
this key shouldn't be in and env variable?

Reply:
sure, no problem